roles/lnls-ans-role-repositories/defaults/main.yml

```diff
@@ -8,10 +8,6 @@ apt_deb_repo_enable_pinning: true
 
 # Set NSLS2 EPICS main debian pacakges pin priority
 epics_deb_main_conf_pin_prio: 500
-# Set NSLS2 EPICS epics-perl pacakges pin priority.
-# Don't make this package installable as it conflicts
-# with other distributions. Use a negative value
-epics_deb_epics_perl_conf_pin_prio: -10
```

Comment:
We shouldn't change anything in the "repositories" role I think. It doesn't install anything. It just sets the "Pin" priority so we can install Debian packages in earlier versions of Ubuntu.

Reply:
ok, if you think epics-perl will ever be used, we can keep role-repositories as it is. done.

Reply:
Thanks. It's just that without this, the host becomes inconsistency and we can't install `epics-perl` in some cases. I think it was on older Ubuntus, because of a `perl` module version clash used by `epics-perl`.